### ISSUE TITLE
Release rollback not updated in Releases list

### ISSUE THREAD

**Issue by jim-docker** [kontena/lens#4589]
When applying a rollback from the Apps -> Releases tab the release in the item-list-layout is not updated

https://user-images.githubusercontent.com/40840436/146572849-69bcb836-3d85-4ab7-89ad-6abb6fa8e946.mov

5.3.3 (linux)

**Comment by jansav** [comment]
UI flow is:

```typescript
await listReleases(...); // current revision = 5 

await rollbackRelease(...); // to revision -> 2

await listReleases(...); // current revision = 5, should be 6

// Note: Just to verify that listReleases eventually return right revision
setTimeout(() => {
  await listReleases(...); // current revision = 6
}, 1500)

```

Back-end issue?

**Comment by jansav** [comment]
> Back-end issue?

Confirmed. Working on fix now.

**Comment by jansav** [comment]
Fixed by #4785
